pkg/sourcereader/embedded.go

```diff
@@ -94,16 +94,8 @@ func (r EmbeddedSourceReader) GetModuleInfo(modPath string, kind string) (module
 		return modulereader.ModuleInfo{}, fmt.Errorf("Source is not valid: %s", modPath)
 	}
 
-	modDir, err := copyFSToTempDir(ModuleFS, modPath)
```

Comment:
I believe ModuleFS and BaseFS (at the top) are no longer used and can be removed from this file. Correct me if I'm mistaken, I haven't checked everywhere.